client/Dockerfile-5.12

```diff
@@ -152,3 +125,27 @@ RUN if [ "$BUILD_QT" != "1" ] ; then echo Download precompiled Qt. && \
     ; fi
 
 ###########################################################################
+
+ENV QTDIR ${QT_ROOT}
+ENV PATH ${QT_ROOT}/bin:${PATH}
+ENV LD_LIBRARY_PATH ${QT_ROOT}/lib/x86_64-linux-gnu:${QT_ROOT}/lib:/usr/local/lib:${LD_LIBRARY_PATH}
+ENV PKG_CONFIG_PATH ${QT_ROOT}/lib/pkgconfig:${PKG_CONFIG_PATH}
+
+# Install QtKeychain
+RUN cd /tmp && \
+    git clone https://github.com/frankosterfeld/qtkeychain.git && \
+    cd qtkeychain && \
+    git checkout v0.10.0 && \
+    mkdir build && \
+    cd build && \
+    cmake ../ && \
+    make -j$(nproc) && \
+    make install && \
+    cd .. && \
+    rm -rf qtkeychain
+
+###########################################################################
+
+# We won't use apt anymore, let's cleanup to shrink the image a bit
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
That won't work. The cleanup needs to happen in the same step as each step is it's own layer. Thus it was always added at the end of the `apt-get install` step.

Reply:
In this case I would just not clean it up or combine all `apt-get` steps 🙈

Reply:
Oh indeed, stupid me. Solved in the new replacement commit.